### PR TITLE
Documentation incorrectly advises to edit node-config.yaml instead of ConfigMaps

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1811348

New steps taken from: https://access.redhat.com/solutions/4830591

### CODE REVIEW

**Comment by mburke5678** [comment]
@sunilcio PTAL

**Comment by sunilcio** [comment]
@mburke5678 LGTM, thanks

File: admin_guide/sysctls.adoc [hunk 1]
```diff
@@ -123,14 +123,71 @@ at-your-own-risk and can lead to severe problems like wrong behavior of
...
 
-. Use the `*kubeletArguments*` field in the *_/etc/origin/node/node-config.yaml_*
-file, as described in
-xref:../admin_guide/manage_nodes.adoc#configuring-node-resources[Configuring Node Resources], to set the desired unsafe sysctls:
+. Add the unsafe sysctls to the `kubeletArguments` parameter of the appropriate xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration map] file, as described in xref:../admin_guide/manage_nodes.adoc#configuring-node-resources[Configuring Node Resources]:
++
+For example:
++
+----
+oc edit cm node-config-compute -n openshift-node
```
Comment (kalexand-rh) [inline admin_guide/sysctls.adoc:131]: $

File: admin_guide/sysctls.adoc [hunk 2]
```diff
@@ -123,14 +123,71 @@ at-your-own-risk and can lead to severe problems like wrong behavior of
...
+....
+    kubeletArguments:
+      allowed-unsafe-sysctls: <1>
+      - "net.ipv4.tcp_keepalive_time"
+      - "net.ipv4.tcp_keepalive_intvl"
+      - "net.ipv4.tcp_keepalive_probes"
+----
+<1> Add the unsafe sysctls you want to use.
+
+. Create a new SCC using the contents of the *restricted* SCC and add the unsafe sysctls:
```
Comment (kalexand-rh) [inline admin_guide/sysctls.adoc:142]: s/using/that uses

File: admin_guide/sysctls.adoc [hunk 3]
```diff
@@ -123,14 +123,71 @@ at-your-own-risk and can lead to severe problems like wrong behavior of
...
+- net.ipv4.tcp_keepalive_probes
+....
+metadata:
+  name: restricted-sysctls <2>
+....
+---- 
+<1> Add the unsafe sysctls you want to use.
+<2> Specify a new name for the SCC.
+
+. Grant the new SCC access to your Pod ServiceAccount.
```
Comment (kalexand-rh) [inline admin_guide/sysctls.adoc:166]: s/ServiceAccount./ServiceAccount:

s/Pod/pod (because 3.x)

File: admin_guide/sysctls.adoc [hunk 4]
```diff
@@ -123,14 +123,71 @@ at-your-own-risk and can lead to severe problems like wrong behavior of
...
++
+----
+$ oc adm policy add-scc-to-user restricted-sysctls -z default -n your_project_name
+----
+
+. Add the unsafe sysctls to the DeploymentConfig for your Pods.
```
Comment (kalexand-rh) [inline admin_guide/sysctls.adoc:172]: s/Pods./pods:

**Review by kalexand-rh — COMMENTED** [review]
Just a few picks, and it LGTM

**Pull request merged**